packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -100,7 +102,7 @@ export class NavBase extends BaseComponent<INavProps, INavState> implements INav
     const classNames = getClassNames(getStyles!, { theme: theme!, className, isOnTop, groups });
 
     return (
-      <FocusZone direction={ FocusZoneDirection.vertical }>
+      <FocusZone direction={ FocusZoneDirection.vertical } ref={ this._focusZone }>
```

Comment:
Again if this is meant to be an exact port over, it should componentRef not just ref